src/SFA.DAS.Commitments.Application/Extensions/RoutingSettingsExtensions.cs

```diff
@@ -10,8 +10,10 @@ public static class RoutingSettingsExtensions
         private const string NotificationsMessageHandler = "SFA.DAS.Notifications.Application";
 
         public static void AddRouting(this RoutingSettings routingSettings)
+
```

Comment:
Hmm, we also seem to have this in `SFA.DAS.CommitmentsV2.Extensions`.

Also, you are taking your whitespacing to the next level.